src/translations/da_DK.json

```diff
@@ -20,7 +20,7 @@
   "ONBOARDING_CONNECT_DD_PAGE_TITLE": "Tilknyt automatisk betaling | Hedvig",
   "COVERAGE_LABEL": "Dækning",
   "COVERAGE_HEADLINE": "Omfattende forsikringsdækning",
-  "COVERAGE_BODY": "Al dækning du mangler lige ved dine fingre. Kundeservice med rigtige mennesker fra 09-22 hver dag. Hurtig forsikringskrav håndteres i appen. Lav selvrisiko på 500 DKK, ingen selvrisiko for store forsikringskrav. Klik på ikonet for mere information.",
+  "COVERAGE_BODY": "Al den dækning du har brug for lige ved hånden. Kundeservice med rigtige mennesker til 22 hver dag. Hurtig håndtering af forsikringskrav i appen. Lav selvrisiko på 500 kr, ingen selvrisiko for store forsikringskrav. Klik på ikonerne nedenfor for mere information.",
```

Comment:
Shouldn't it still say that it's from 9 AM?

Reply:
They changed this back and forth since it's 9 on weekdays and from 10 on weekends :)

Reply:
But it might be that they have changed it again 😁

Reply:
Haha, OK, let's leave it like this then 😄